### PR TITLE
README Improvements

### PR DESCRIPTION
**Pull request by DanRyanIrish**
@Cadair, here is an attempt to improve the README.  Hopefully it's already along the lines you were thinking.  If not, perhaps you can plagiarise some of it in your attempt to clean it up.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,8 +1,7 @@
 `ndcube`
 ========
 
-|Latest Version| |Build Status| |Build status| |codecov| |Research
-software impact| |Powered by NumFOCUS| |Powered by Sunpy|
+|Latest Version| |Build Status| |Build status| |codecov| |Research software impact| |Powered by NumFOCUS| |Powered by Sunpy|
```
Comment (DanRyanIrish) [inline README.rst]: @Cadair @nabobalis The ```|Research software impact|``` isn't actually used below which I think is why the tag is "broken".  Where should it be pointing?  Or should we just remove it?

Comment (nabobalis) [reply README.rst]: I removed the URL for the badge but forgot to remove it. I'd delete it.

File: README.rst [hunk 2]
```diff
@@ -17,60 +16,138 @@ software impact| |Powered by NumFOCUS| |Powered by Sunpy|
...
+::
+   $ git pull upstream master
+
+To push any changes you make to your github account by doing:
+::
+   $ git push origin branch-name
+where ``branch-name`` is the name of the branch you're working on.  Then
+from your github account you can request your changes to be merged to
+the main repo.  For more information on on git versioning control,
+github, and issueing pull requests, see ??????
```
Comment (DanRyanIrish) [inline README.rst]: @Cadair , @nabobalis , is there a SunPy-recommended resource on git and github that we can direct people to here?

Comment (nabobalis) [reply README.rst]: Try this: http://docs.sunpy.org/en/stable/dev_guide/version_control.html

Comment (DanRyanIrish) [reply README.rst]: Perfect!

**Pull request merged**